tests/fixtures/resources.py

```diff
@@ -6,11 +6,15 @@
 
 @pytest.fixture
 def resource0(project0):
-    return Resource.objects.create(
-        project=project0, path="resource0.po", format="po")
+    return Resource.objects.get(project=project0, path="resource0.po")
 
 
 @pytest.fixture
 def resource1(project1):
+    return Resource.objects.get(project=project1, path="resource1.po")
+
+
+@pytest.fixture
+def resourceX(projectX):
     return Resource.objects.create(
```

Comment:
Why is this a `create` still and not a `get` like the others? You are creating `projectX` in the Environment.

Reply:
because any fixture with `X` at end has minimal setup in the env, so in this case its not already created

eg. userX has no contributions and resourceX has no entities, etc

Reply:
actually, `resourceX` was a bad example, because `projectX` has no resources, and therefore `resourceX` doesnt exist